src/SupabaseClient.ts

```diff
@@ -88,6 +89,7 @@ export default class SupabaseClient<
       realtime: DEFAULT_REALTIME_OPTIONS,
       auth: { ...DEFAULT_AUTH_OPTIONS, storageKey: defaultStorageKey },
       global: DEFAULT_GLOBAL_OPTIONS,
+      postgrest: DEFAULT_POSTGREST_OPTIONS,
```

Comment:
There already is a `settings.db` project, although that targets database access-related modifiers.
This addition simply targets how the lib interfaces with PostgREST